app/controllers/landing_page_controller.rb

```diff
@@ -1,21 +1,8 @@
 class LandingPageController < ApplicationController
   def show
-    @form = SearchForm.new
-  end
+    @page_title = t('landing_page.title_tag')
+    @meta_tags = t('landing_page.meta_tag')
```

Comment:
So this feels a bit off for me, usually you'd want to only pass one instance variable to the view (though, that's not always feasible). In this scenario, I writing a helper method might be a better approach.